bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.regex.Pattern;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class TestBigtableHBaseVersion {
+
+  @Test
+  public void testVersionNumber() {
+    // This patterns matches string with "1.14+" versions, The version must have 3 parts i.e. x.y.z
+    Pattern versionPattern =
+        Pattern.compile("^(([1]\\.([1][4-9]|[2-9]\\d))|([2-9]\\.\\d{1,2}))\\.");
```

Comment:
It would be nicer/easier to debug if  you extract the numbers with this regex: `"^(\\d+)\\.(\\d+)\\.(\\d+)"`
Then do a comparison separately.
```java
int result = ComparisonChain.start()
  .compare(1, Integer.parse(matcher.group(1))
  .compare(14, Integer.parse(matcher.group(2))
  .compare(0, Integer.parse(matcher.group(3))
  .result();

assertTrue(result <= 0, "Expected BigtableVersion.getVersion() to be at least 1.14.0")
```

Reply:
Thanks for the detailed suggestion!!
Have updated the test case to check with "1.14.1" as minimum version(given current snapshot version is 1.14.1-SNAPSHOT)